src/app/button-more/button-more.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
please check that all tests are passing -- if no -- just delete them. Actually we talked about it today and decided to delete them for now